src/FileInput/FileInput.tsx

```diff
@@ -110,6 +130,7 @@ const propTypes = {
   store: PropTypes.func.isRequired,
   accept: PropTypes.string,
   size: PropTypes.string,
+  isMessagingAttachmentInput: PropTypes.bool,
```

Comment:
Another thought here: wondering if there's an opportunity to potentially abstract this prop into something non-messaging related e.g. some kind of icon override since we're mainly checking this boolean so we can update the icon on line 203